MetaMask/metamask-extension
# Installing design tokens and writing docs in storybook

Fixes: #13564

Explanation: Installing [`@metamask/design-tokens`](https://github.com/MetaMask/design-tokens) repo and adding guidelines on how to use colors.

- Docs could be improved by giving color swatches and descriptions I'll update this once we expose the `token.json` from the design-tokens repo.

Manual testing steps: 
  - Check latest [storybook build](https://491729-42009758-gh.circle-artifacts.com/0/storybook/index.html?path=/story/design-tokens-color--page) Color docs page
 **OR**
  - run `yarn setup && yarn storybook`
  - Go to http://localhost:6006/?path=/story/design-tokens-color--page&globals=measureEnabled:true
  - See Color Documentation
  - Go to component story
  - Open developer tools
  - See CSS variables are available to use in styles

## Screenshots

CSS variables available to use in styles

<img width="1440" alt="Screen Shot 2022-02-17 at 12 48 00 PM" src="https://user-images.githubusercontent.com/8112138/154392024-e134f251-e6e3-48d6-9022-5d9318fe2d7f.png">

Color Documentation

![screencapture-localhost-6006-2022-02-23-07_15_09](https://user-images.githubusercontent.com/8112138/155218440-3d57eca1-b29f-45ff-b70e-f5b82ccc377c.png)
